Sources/FTPropertyWrappers/Serialized.swift

```diff
@@ -14,15 +14,34 @@ public final class Serialized<Value> {
         }
     }
 
-    /// Did set observer for stored property. Notice, that didSet event is called on the synchronization queue. You should free this thread asap with async call, since complex operations would slow down sync access to the property.
+
+    /// Did set observer for stored property. Notice, that didSet event is called on the synchronization queue.
+    /// You should free this thread asap with async call, since complex operations would slow down sync access
+    /// to the property.
     public var didSet: ((Value) -> Void)?
 
     /// Inserting initial value to the property. Notice, that this operation is NOT DONE on the synchronization queue.
+    ///  This initializer uses predefined label for dispatch queue.
+    /// - Parameter wrappedValue: Default value, not inserted on synchronization queue
     public init(wrappedValue: Value) {
+        queue = DispatchQueue(label: "app.futured.ftpropertywrappers.serialized")
+        value = wrappedValue
+    }
+
+
+    /// Initializes the property wrapper. Default value is NOT inserted on synchronization queue. This initializer
+    /// allows to specify custom queue name. This initializer must be overloaded. If we use default value in
+    /// initializer, compiler can not syntetize default constructor for enclosing types that accepts generic type
+    /// argument of Serialized.
+    /// - Parameters:
+    ///   - wrappedValue: Default value, not inserted on synchronization queue
+    ///   - label: Label for synchronization queue.
+    public init(wrappedValue: Value, customQueue label: String) {
```

Comment:
I am still missing init where you can pass `DispatchQueue`. Initializing using label can be insufficient in some cases.

```swift
public init(wrappedValue: Value, queue: DispatchQueue) {
```